webapp/shop/cred/views.py

```diff
@@ -351,6 +373,13 @@ def cred_assessments(trueability_api, **_):
 
 @shop_decorator(area="cred", permission="user", response="html")
 def cred_exam(trueability_api, **_):
+    email = flask.session["openid"]["email"].lower()
+    if (
+        not flask.current_app.debug
```

Comment:
Would not have left `not flask.current_app.debug` after I finished developing/testing.